go/teams/member_set.go

```diff
@@ -265,7 +297,18 @@ func (m *memberSet) AddRemainingRecipients(ctx context.Context, g *libkb.GlobalC
 		if _, ok := m.recipients[uv]; ok {
 			continue
 		}
-		if _, err := m.loadMember(ctx, g, uv, true, forceUserPoll); err != nil {
+		if _, ok := m.botRecipients[uv]; ok {
+			continue
+		}
+
+		var storeMemberKind storeMemberKind
+		if _, ok := existingBots[uv]; ok {
+			storeMemberKind = storeMemberKindBotRecipient
+		} else {
+			storeMemberKind = storeMemberKindRecipient
+		}
```

Comment:
god forbid some user be both bot and not would you rather they get non-bot here?